cmd/manager/main.go

```diff
@@ -6,7 +6,6 @@ import (
 	"github.com/mongodb/mongodb-kubernetes-operator/pkg/controller"
 	"go.uber.org/zap"
 	"os"
-
```

Comment:
when are we getting the commit hook for gofmt and goimports?

![](https://media.giphy.com/media/Y0mr4v1uBtIwE/giphy.gif)

Reply:
@rodrigovalin  is working on it now I believe!

Reply:
I am, doing it right now